dist/accordion/css/accordion.min.css

```
@@ -1 +1,62 @@
-.aria-accordion__panel[hidden]{display:none}
\ No newline at end of file
+/* For legacy browsers support */
+.aria-accordion__panel[hidden] {
+  display: none;
+}
+
+/* For accordion panels with a transition effect */
+.aria-accordion__panel--transition {
+  display: block;
+  max-height: 100vh;
+  visibility: visible;
+  -webkit-transition: max-height 800ms ease-in;
+  -moz-transition: max-height 800ms ease-in;
+  -ms-transition: max-height 800ms ease-in;
+  -o-transition: max-height 800ms ease-in;
+  transition: max-height 800ms ease-in;
+  overflow: visible;
```

```suggestion
  overflow: auto;
```
We need a scrollbar in case of very very long panel content.